tracer/test/snapshots/DataStreamsMonitoringTests.SubmitsDataStreams.verified.txt

```diff
@@ -0,0 +1,218 @@
+﻿{
+  Env: integration_tests,
+  Service: Samples.DataStreams.Kafka,
+  TracerVersion: 2.x.x.x,
+  Lang: dotnet,
+  Stats: [
+    {
+      Start: 1661520120000000000,
+      Duration: 10000000000,
+      Stats: [
+        {
+          EdgeTags: [
+            type:internal
+          ],
+          Hash: 12926600137239154356,
+          PathwayLatency: /w==,
+          EdgeLatency: /w==,
```

Comment:
Is this the scrubbed `0xFF` that represents sketch bytes? I'm guessing so, but it seems like gibberish to me 😕

Reply:
> Is this the scrubbed 0xFF that represents sketch bytes? 

Yep, exactly.

> it seems like gibberish to me

It's what you get if you use `System.Text.Json` to serialize a byte array - it base64 encodes it 🙂 e.g.

```csharp
Console.WriteLine(System.Text.Json.JsonSerializer.Serialize(new byte[]{ 0xFF})); // prints /w==
```